types/lib/query-interface.d.ts

```diff
@@ -8,14 +8,17 @@ import {
   WhereOptions,
   Filterable,
   Poolable,
-  ModelCtor, ModelStatic, ModelType
```

Comment:
ModelCtor is removed here because of an earlier PR, right?

Reply:
It's the same as ModelStatic now but in this specific case it just was not used, it was added (but not used) in https://github.com/sequelize/sequelize/pull/12405